include/event_scripts.h

```diff
@@ -28,6 +28,9 @@ extern const u8 EventScript_FollowerFaceUp[];
 extern const u8 EventScript_FollowerFaceResult[];
 extern const u8 EnterPokeballMovement[];
 
+extern const u8 Common_Movement_FollowerSafeStart[];
+extern const u8 Common_Movement_FollowerSafeEnd[];
```

Comment:
I feel like we should/could check this with merrp and maybe PR it back to her branch too? A 600 line jump is quite a lot, so I just want to make sure we don't accidentally break followers with this

Reply:
Eh, I would rather revert the change than get merrp involved with something so trivial.

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5044/commits/0a997fa6ca84dbb2c810eb5bcc08f3adcdeb9259